services/management/ia/rules_service_test.go

```diff
@@ -146,6 +148,148 @@ groups:
 		assert.Equal(t, expected, string(file))
 	})
 
+	t.Run("wrong parameter", func(t *testing.T) {
+		testDir, err := ioutil.TempDir("", "")
+		require.NoError(t, err)
+		t.Cleanup(func() {
+			err = os.RemoveAll(testDir)
+			require.NoError(t, err)
+		})
+
+		// Create test rule
+		rules := NewRulesService(db, templates, vmAlert, alertManager)
+		rules.rulesPath = testDir
+		_, err = rules.CreateAlertRule(context.Background(), &iav1beta1.CreateAlertRuleRequest{
+			TemplateName: "test_template",
+			Disabled:     false,
+			Summary:      "some testing rule",
+			Params: []*iav1beta1.RuleParam{{
+				Name: "unknown parameter",
+				Type: iav1beta1.ParamType_FLOAT,
+				Value: &iav1beta1.RuleParam_Float{
+					Float: 1.22,
+				},
+			}},
+			For:      durationpb.New(2 * time.Second),
+			Severity: managementpb.Severity_SEVERITY_INFO,
+			CustomLabels: map[string]string{
+				"baz": "faz",
+			},
+			Filters: []*iav1beta1.Filter{{
+				Type:  iav1beta1.FilterType_EQUAL,
+				Key:   "some_key",
+				Value: "'60'",
+			}},
+			ChannelIds: []string{channelID},
+		})
+		tests.AssertGRPCError(t, status.New(codes.InvalidArgument, "Unknown parameters [unknown parameter]."), err)
+	})
+
+	// t.Run("missing parameter", func(t *testing.T) { TODO: PMM-7279 fix once templates will support parameters without default value.
```

Comment:
let's use `t.Skip()` in such cases, so it will be visible in our tests list.

Reply:
Done